mlrun/render.py

```diff
@@ -370,12 +405,15 @@ def time_str(x):
         df.drop("labels", axis=1, inplace=True)
         df.drop("inputs", axis=1, inplace=True)
         df.drop("artifacts", axis=1, inplace=True)
+        df.drop("output_path", axis=1, inplace=True)
     else:
         df["labels"] = df["labels"].apply(dict_html)
         df["inputs"] = df["inputs"].apply(inputs_html)
-        df["artifacts"] = df["artifacts"].apply(
-            lambda x: artifacts_html(x, "target_path")
+        df["artifacts"] = df[["artifacts", "output_path"]].apply(
+            lambda artifacts_tuple: artifacts_html(artifacts_tuple, "target_path"),
+            axis=1,
         )
+        df.drop("output_path", axis=1, inplace=True)
```

Comment:
why

Reply:
because I don't want it to be part of the html that is presented in the jupy (it is being displayed late in get_tblframe)

Reply:
added a comment